validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V525Spec extends LarEditCheckSpec with BadValueUtils {
+  property("HOEPA Type must = 1, or 2") {
```

Comment:
Where does the word `Type` come from? Given that it is a loaded term in Scala for many reasons I hesitate to use it unless there is a good reason. The edits document uses HOEPA `status`